modules/service/src/main/scala/lucuma/odb/graphql/mapping/QueryMapping.scala

```diff
@@ -274,10 +275,11 @@ trait QueryMapping[F[_]] extends Predicates[F] {
         Select("target", Nil,
           Unique(
             Filter(
-              And(
+              and(List(
                 Predicates.target.id.eql(pid),
                 Predicates.target.program.isVisibleTo(user),
-              ),
```

Comment:
My understanding was that the fact that guide stars are stored in the target table is purely an internal implementation thing. Since guide stars are very different in that they don't have an existence separate from the observation they belong to, I was planning on never exposing the target id for guide stars to the API.

Reply:
Does this mean that we'll need to add controls for skipping to the "next" highest ranked guide star?

Reply:
Yes, I think the main thing is that by default when you ask for an observation's targets you probably just want the science targets so that's what you should get.  Somehow though we'll need to be able to see what the guide star / blind offset / etc is in the UI. The `target` and `targets` queries, properly restricted in the `WHERE` clause, is one simple (?) way of getting this but maybe there are better options.

I think it is fine to merge this PR now though and worry about the rest later.